src/app/components/Video/index.jsx

```diff
@@ -1,30 +1,62 @@
 import React from 'react';
+import styled from 'styled-components';
+import Helmet from 'react-helmet';
 import { videoComponentPropTypes } from '../../models/propTypes';
 
+const MediaPlayerContainer = styled.div`
+  height: 26em;
+  width: 100%;
+`;
+
 const Video = ({
-  pid,
-  kind,
+  id,
   title,
-  items,
-  holdingImageUrl,
   statsAppName,
   statsAppType,
   statsCountername,
   statsDestination,
   uiLocale,
+  mediaPlayerSettings,
 }) => (
-  <div>
-    <div>video pid: {pid}</div>
-    <div>kind: {kind}</div>
+  <>
+    <Helmet>
+      <script type="text/javascript">
+        {`
+          function mediaPlayerSetup(container) {
+            require(['bump-4'], (bump) => {
+              var mediaPlayer = bump.player(
+                document.getElementById('${id}'),
+                ${JSON.stringify(mediaPlayerSettings)});
+              mediaPlayer.load();
+            });
+          }
+        `}
+      </script>
+      <script type="text/javascript">
+        {`
+          function initialiseRequires() {
+            var requiredScripts = {
+              "bump-4": "https://emp.bbci.co.uk/emp/bump-4/bump-4",
```

Comment:
Should this URL vary based on environment, as it does on Cookbook?

Reply:
@rhell I was told the versions of bump are the test/live versions of the SMP player, so I assumed we don't need to differ them by platform?